audio.py

```diff
@@ -30,9 +35,32 @@ def read_audio(
 		elif audio_path.endswith('.wav'):
 			sample_rate_, signal = scipy.io.wavfile.read(audio_path)
 			signal = torch.as_tensor(signal[None, :] if len(signal.shape) == 1 else signal.T)
+		elif backend == 'soundfile' and audio_path.endswith('.gsm'):
+			signal, sample_rate_ = soundfile.read(audio_path, dtype = 'float32')
```

Comment:
For better comparability with other backends and simplicity, let's pass `dtype = "int16"` instead. Later we can test whether it's faster to do the int16->float32 conversion inside the backend or if it's okay to always do the conversion in the postproc code.

Reply:
done